rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -958,7 +961,8 @@ void MasterGraph::output_routine()
             }
             _rb_block_if_full_time.start();
             // _ring_buffer.get_write_buffers() is blocking and blocks here until user uses processed image by calling run() and frees space in the ring_buffer
-            auto write_buffers = _ring_buffer.get_write_buffers();
+            auto write_output_buffers = _ring_buffer.get_write_buffers();
```

Comment:
Can you call _ring_buffer.get_write_buffers().first to get the write_output_buffers?